cmd/frostfs-adm/internal/modules/morph/group.go

```diff
@@ -53,12 +53,11 @@ func openContractWallet(v *viper.Viper, cmd *cobra.Command, walletDir string) (*
 	p := filepath.Join(walletDir, contractWalletFilename)
 	w, err := wallet.NewWalletFromFile(p)
 	if err != nil {
-		if !os.IsNotExist(err) {
-			return nil, fmt.Errorf("can't open wallet: %w", err)
+		if os.IsNotExist(err) {
+			cmd.Printf("Contract group wallet is missing, initialize at %s\n", p)
+			return initializeContractWallet(v, walletDir)
 		}
-
-		cmd.Printf("Contract group wallet is missing, initialize at %s\n", p)
-		return initializeContractWallet(v, walletDir)
+		return nil, fmt.Errorf("can't open wallet: %w", err)
```

Comment:
Can we remove these lines from the diff, please? Seem identical.

Reply:
fixed